documentation/staging/content/quickstart/create-domain.md

```diff
@@ -22,7 +22,12 @@ weight: 6
     with the value specified by the `-d` flag.  For example, the command above would create a secret named
     `sample-domain1-weblogic-credentials`.
 
-1.	Create a new image with a domain home by running the [create-domain](http://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script. First, copy the sample [create-domain-inputs.yaml](http://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml) file and update your copy with:  
+1.	Create a new image with a domain home by running the [create-domain](http://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh) script.
+
+    {{% notice tip %}} For a detailed understanding of the steps that the `create-domain.sh` script performs for you, see the bulleted items under [Use the script to create a domain]({{< relref "/samples/simple/domains/domain-home-in-image/#use-the-script-to-create-a-domain" >}}).
+    {{% /notice%}}
```

Comment:
IMO we either want to strongly encourage customers to read the 'detailed steps' so that they understand what the script is doing for them, or we should summarize its steps directly here. We should also take the opportunity to emphasize that the script is only meant for demonstration purposes.

How about:

```
1. In this step, we will setup and run the sample 
[create-domain.sh](http://github.com/oracle/weblogic-kubernetes-operator/blob/main/kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain.sh)
script. 

The script's behavior is controlled by an 'inputs file' plus command line options. The script 
will download the WebLogic Image Tool and WebLogic Deploy Tool, use these tools to create
a new image with a domain home, create a domain resource YAML file that references the image, 
and, finally, if the `-e` option is specified, deploy the domain resource to Kubernetes.

{{% notice tip %}} The script is for demonstration purposes only. We recommend that 
production users directly leverage the WebLogic Image Tool and directly work with domain 
resource files. For a detailed understanding of the steps that the `create-domain.sh` script 
performs for you, see the bulleted items under [Use the script to create a domain]({{< relref "/samples/simple/domains/domain-home-in-image/#use-the-script-to-create-a-domain" >}}). 
{{% /notice%}}
```

BTW, IMO we should eventually make sure that all of the samples that use the create-domain script have similar wording as ^^^.

Reply:
I guess we are going to agree to disagree on these points: 1. "We should summarize its steps directly here." Not done for myriad reasons. Period. 2. "Strongly encourage customers to read the 'detailed steps' so that they understand what the script is doing for them (not just provide a link)" IMO: Overkill. This is the **Quick Start**. We make detailed info available for those who want to know it, read it, understand it. The x-ref is good enough. 3. "We should also take the opportunity to emphasize that the script is only meant for demonstration purposes." Everything in the Quick Start is for demonstration purposes only, hence, the first line in the QS Overview, "Use this Quick Start guide to create a WebLogic Server deployment in a Kubernetes cluster with the Oracle WebLogic Server Kubernetes Operator. Please note that this walk-through is for demonstration purposes only, not for use in production." I believe that a create-domain script deep dive is more appropriate where it is currently located, and perhaps, in other samples that use it. However, as always, my mind is open to other opinions.

Reply:
I don't agree.

The intent of Quick Start is teach new users about operator usage. If new users do not understand the steps Quick Start is performing for them, then they will fail to understand operator usage. 

We have seen this happen frequently in practice. For example, we have seen users not grasp that a 'domain resource' file even exists because they naturally assume the sample's 'inputs' file is the most important file and because the create-domain.sh script silently and implicitly creates and deploys the domain resource file. But, we want customers to understand that the 'domain resource' is the central jewel upon which the entire Operator project turns...

We have also frequently seen customers mistakenly assume that 'create-domain.sh' is a critical part of the Operator usage, and even assume that they will always need to use it.

For these reasons, Monica and Ryan (and others) have stated we should eventually replace Quick Start with a far more detailed 'step by step' approach.  This hasn't been done due to competing priorities, and the added wording I'm suggesting is a compromise.

Reply:
I certainly understand and agree with the points you make here about the Quick Start and new users understanding operator functioning. If it needs to be replaced, modified, or enhanced, then it should be. However, I don't see how your wording is a compromise (about any of it). We all understand that the QS is meant to be a quick and easy way to get the operator up and running, simple steps to follow. Deep dives and emphasis on what users need to know about what's under the covers should come from the samples. Pointers in the QS help new users find the deeper explanations. If we need to downplay the create-domain script, we can add that wording, but your idea was to ask them/strongly recommend that they know its details. Why, if it is not a critical part of operator usage, and they will NOT always need to use it? To summarize my points: 1. If the Quick Start is not doing a good job of getting new users up and running with the operator, lets change it. 2. Put the detailed information/best practices/critical knowledge for operator usage in the samples and we'll point to it as the next level of learning for new users.

Reply:
Agreed on parts 1 & 2. There are long standing plans for a full solution which will require a substantial amount of work.  

IMO, the small amount of additional wording ^^^ will go a long way towards helping ensure that the sample's users learn the right lessons in the mean-time. The wording guide's sample users away from common misunderstandings that we've seen on multiple occasions during support, and it also teaches users what the temporary throw-away helper script is doing for them.

Reply:
I also want to update the quick start to be more like the MII sample, but I don't understand holding up a simple improvement for work that will be substantially longer running.

Reply:
Let's not complicate simple fixes.  Take this debate offline and file an additional Jira/issue as needed.

Reply:
Latest updates LGTM - with one minor comment remaining. I'll start a new thread for that comment.

Reply:
ok.